wgengine/tsdns/tsdns.go

```diff
@@ -209,6 +209,17 @@ func (r *Resolver) Resolve(domain string, tp dns.Type) (netaddr.IP, dns.RCode, e
 		return netaddr.IP{}, dns.RCodeNameError, nil
 	}
 
+	// For everything except for a few types that are explictly not implemented, no records are
+	// returned. This is what every other DNS system I could find does: always return NOERROR
+	// without any records whenever the requested record type is unknown. You can try this with:
+	// dig -t TYPE9824 example.com
+	// and note that NOERROR is returned, despite that record type being made up.
+	var explictlyUnimplemented =
```

Comment:
Once https://github.com/tailscale/tailscale/pull/1098 is in, these cases (and accompanying comments) can be folded into the switch statement below. That'll help readability.